.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,37 @@
+name: 🚀 Feature request
+description: Submit a proposal for a new PyTorch tutorial or improvement of an existing tutorial
+title: "💡 [REQUEST] - <title>"
+labels: [
+  "feature"
+]
+
+body:
+- type: textarea
+  attributes:
+    label: 🚀 Descirbe the improvement or the new tutorial
+    description: |
+      **Describe the improvement**
+    placeholder: |
+      Explain why this improvement or new tutorial is important. For example, *"This tutorial will help users to better understand feature X of PyTorch."* If there is a tutorial that you propose to replace, add here. If this is related to another GitHub issue, add a link here.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Existing tutorials on this topic
```

Comment:
What's the point of linking to existing tutorials?

Putting on my "contributor" hat, if I thought such tutorials already existed I'd prob not have bothered creating this new one

Reply:
This is if someone is proposing a tutorial on a topic that is already covered but might be outdated, uses old tech and so on. Like a better version of the existing one.